### DISCUSSION TITLE
Storybooks actions working in development environment but not working in production environment.

### DISCUSSION THREAD

**Discussion by vvs09** [storybookjs/storybook#25204] [Help]
Storybook Version 7.4.6
Angular 15.1.4

**Comment by vvs09** [comment]
Error in the browser when in production mode Type error click is not a function.

**Comment by Marklb** [comment]
Can you provide a repro? I created a project on Angular 15.1.4 and Storybook 7.4.6 and actions seem fine in my static build.

**Reply by vvs09** [reply]
Did it still if you set enableProdMode= true in your angular.json?

**Reply by Marklb** [reply]
Are you asking if I put `"enableProdMode": true` in the storybook build options? I did not before and just assumed it was a prod build, because the `window.ng` tools were not available, but I just tried again, with the following and the action still seems fine
```json
"build-storybook": {
  "builder": "@storybook/angular:build-storybook",
  "options": {
    "configDir": ".storybook",
    "browserTarget": "discussion-25204:build",
    "compodoc": true,
    "compodocArgs": [
      "-e",
      "json",
      "-d",
      "."
    ],
    "outputDir": "storybook-static",
    "enableProdMode": true
  }
}
```

**Reply by vvs09** [reply]
I'm sorry for the late response but here's a repo(https://github.com/vvs09/Storybookv7-angular-material/blob/main/storybook/src/stories/paginator.stories.ts). if you set "enableProdMode": false the pagination actions works fine but when set enableProdMode:true the actions for the paginator does not work. Thank you.

**Reply by pbintcha** [reply]
I was also facing the same issue, set enableProdMode to false works also
